adoc/reference/reference.adoc

```diff
@@ -297,9 +297,19 @@ Arkane also offers whitelabel wallets. link:whitelabel.html[Go to documentation]
 Signs arbitrary data using a specific account.
 
 .URL
-https://connect-staging.arkane.network/signatures
+https://connect-staging.arkane.network/signature/execute
```

Comment:
Shouldn't this be .../signature**s**/execute

Reply:
this is actually not implemented yet, tis is for connect so it will depend on how it is implemented in the frontend and it is currently using /signature/execute

Reply:
allee dan